### PR TITLE
Remove survey test schemas and associated integration tests

### PR DESCRIPTION
**Pull request by jonnyshaw89**
### What is the context of this PR?
Remove real surveys that are being used in tests. Refactor tests to use test schemas to maintain coverage

### How to review 
All real schemas and starwars should be removed.
Where tests have been removed, other scenarios should be present

### Checklist

* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

File: tests/integration/rsi/test_rsi_submission_data.py [hunk 1]
```diff
@@ -1,178 +0,0 @@
-from tests.integration.integration_test_case import IntegrationTestCase
-
-
-class TestRsiSubmissionData(IntegrationTestCase):
-
-    def test_submission_data_1_0102(self):
```
Comment (iwootten) [inline tests/integration/rsi/test_rsi_submission_data.py:6]: Do we need a replacement test to cover submission data? Looks like it was only tested as part of rsi/mci.

File: app/submitter/convert_payload_0_0_1.py [hunk 2]
```diff
@@ -80,8 +80,6 @@ def _get_checkbox_answer_data(answer_store, answer_schema, value):
 
     if all(q_code is not None for (q_code, _) in qcodes_and_values):
         checkbox_answer_data.update(qcodes_and_values)
-    else:
```
Comment (iwootten) [inline app/submitter/convert_payload_0_0_1.py]: How come this has been changed?

File: tests/integration/rsi/test_rsi_submission_data.py (hunk 1 above)
Comment (jonnyshaw89) [reply tests/integration/rsi/test_rsi_submission_data.py:6]: It should be tested as part of `test_convert_payload_0_0_1.py`

File: app/submitter/convert_payload_0_0_1.py (hunk 2 above)
Comment (jonnyshaw89) [reply app/submitter/convert_payload_0_0_1.py]: I have added this back with a test

**Review by bitdivision — APPROVED** [review]
:+1:

**Comment by jonnyshaw89** [comment]
@iwootten can you re-review

**Review by iwootten — APPROVED** [review]

**Pull request merged**